### ISSUE TITLE
Apache container issue...

### ISSUE THREAD

**Issue by foxhoundv** [nextcloud/all-in-one#1784]
### Steps to reproduce
1. Unknown

### Expected behavior 
Nextcloud to run normally

### Actual behavior 
I was using Nextcloud fine for 8 days, Was using it this morning and then just before noon I went to go to nextcloud and I received a 503 error in my browser. I checked the container and there was this error over and over...
{"level":"error","ts":1673981156.042249,"logger":"http.handlers.reverse_proxy","msg":"aborting with incomplete response","error":"context canceled"}

### Host OS 
Ubuntu 20.04.5 LTS

#### Nextcloud AIO version 
v4.1.0 (now I updated stopped all containers and created backups. I was on 4.0.1)

#### Current channel 
latest

#### Other valuable info 
The admin AIO panel shows this for the apache container...
![image](https://user-images.githubusercontent.com/16512250/212986533-9e85dd90-e9ba-4b5c-a161-685ecb83732c.png)

**Comment by Zoey2936** [comment]
do you use an reverse proxy?

**Comment by foxhoundv** [comment]
Yes and haven't changed anything with it. I have been using AIO for 7 months and it has worked fine.

**Comment by foxhoundv** [comment]
I checked with some settings in my reverse proxy and changed my health checks to basic and now I get this...
![image](https://user-images.githubusercontent.com/16512250/212989433-50285fb9-785e-41e7-b9d9-ccb862139edf.png)

**Comment by szaimen** [comment]
Can you try to restore the last backup, restart the containers another time and check if that makes things work again?

It looks like one of your installed apps might interfer here...

**Comment by szaimen** [comment]
If you get into the same state again, please post the nextcloud container logs here. Thanks!

**Comment by foxhoundv** [comment]
Waiting for database to start...
              now              
-------------------------------
 2023-01-17 12:18:19.788449-07
(1 row)

+ '[' -f /dev-dri-group-was-added ']'
++ find /dev -maxdepth 1 -mindepth 1 -name dri
+ '[' -n '' ']'
+ set +x
Installing imagemagick via apk...
Enabling Imagick...
Configuring Redis as session handler...
Setting php max children...
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value tempdirectory set to string /mnt/ncdata/tmp/
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Applying one-click-instance settings...
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value one-click-instance set to boolean true
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value one-click-instance.user-limit set to integer 100
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value one-click-instance.link set to string https://nextcloud.com/all-in-one/
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
support already enabled
Adjusting log files...
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value logfile set to string /var/www/html/data/nextcloud.log
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Config value logfile for app admin_audit set to /var/www/html/data/audit.log
Applying network settings...
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value trusted_domains => 1 set to string nc2.fruitlandelectric.com
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value overwrite.cli.url set to string https://nc2.fruitlandelectric.com/
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value htaccess.RewriteBase set to string /
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
.htaccess has been updated
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value files_external_allow_create_new_local set to boolean false
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value trusted_proxies => 0 set to string 127.0.0.1
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value trusted_proxies => 1 set to string ::1
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Config value base_endpoint for app notify_push set to https://nc2.fruitlandelectric.com/push
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value onlyoffice => jwt_secret set to string a6ff353536debd99f36cfe32245386c4a071e8fe41692127
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value onlyoffice => jwt_header set to string AuthorizationJwt
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Config value DocumentServerUrl for app onlyoffice set to https://nc2.fruitlandelectric.com/onlyoffice
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value allow_local_remote_servers set to boolean true
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade

                                                               
  There are no commands defined in the "talk:turn" namespace.  
                                                               

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade

                                                               
  There are no commands defined in the "talk:turn" namespace.  
                                                               

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade

                                                               
  There are no commands defined in the "talk:stun" namespace.  
                                                               

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade

                                                                    
  There are no commands defined in the "talk:signaling" namespace.  
                                                                    

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade

                                                                    
  There are no commands defined in the "talk:signaling" namespace.  
                                                                    

Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value enabledPreviewProviders => 0 set to string OC\Preview\Imaginary
Nextcloud or one of the apps require upgrade - only a limited number of commands are available
You may use your browser or the occ upgrade command to do the upgrade
System config value preview_imaginary_url set to string http://nextcloud-aio-imaginary:9000
[17-Jan-2023 12:19:17] NOTICE: fpm is running, pid 341
[17-Jan-2023 12:19:17] NOTICE: ready to handle connections

**Comment by szaimen** [comment]
Thanks, but is this after restoring from backup?

**Comment by szaimen** [comment]
Anyway, since there doesn't seem to be a hint which app this causes, you can probably remove the update hint with:

`sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ upgrade`

Afterwards should a restart probably be a good idea.

**Comment by szaimen** [comment]
Hi, I just created this Wiki article: https://github.com/nextcloud/all-in-one/discussions/1787

**Comment by szaimen** [comment]
Moving to discussions since I cannot reproduce the issue. It likely got triggered by an app update in Nextcloud. The only real solution is installing less apps that potentially could trigger this.
